.github/ISSUE_TEMPLATE/sweep-template.yml

```diff
@@ -0,0 +1,22 @@
+name: Sweep Issue
```

Comment:
how ignore it?

Reply:
I think we can just drop it in a separate PR